compile.sh

```diff
@@ -0,0 +1,7 @@
+
```

Comment:
Don’t have to change it, but I’m curious re: the blank lines at the top of this and the Makefile.

Also, no shebang here – is that kosher?